src/components/swap-history/index.tsx

```diff
@@ -112,14 +112,45 @@ const Transaction: FC<{ address: string; transaction: TransactionProps }> = ({
 const Component: FC = () => {
   const { t } = useTranslation();
   const { address = "", isConnected } = useAccount();
-  const { transactions } = useSwapHistory();
+  const { transactions, clearHistory } = useSwapHistory();
 
   return isConnected ? (
     <div className="swap-history">
-      <span className="heading">{t(constantKeys.TRANSACTIONS)}</span>
-      {transactions.map((transaction, index) => (
-        <Transaction key={index} address={address} transaction={transaction} />
-      ))}
+      <div className="history-header">
+        <span className="heading">{t(constantKeys.TRANSACTIONS)}</span>
+        {transactions.length > 0 && (
+          <button 
+            className="clear-history-btn" 
+            onClick={clearHistory}
+            aria-label="Clear transaction history"
+          >
+            <svg 
```

Comment:
Lets store icons in https://github.com/vultisig/launch-web/blob/swap-page-improvements/src/icons/index.tsx to be able to reuse them in the future and also make the code easier to maintain